tasks/modules/logging/syslog.rb

```diff
@@ -2,9 +2,35 @@
 
 module Logging
   module Syslog
-    def self.log message
+    def self.log message, level = nil
+
+      # Tumbler Collin severity mapping:
+      # - http://www.rubydoc.info/gems/collins_client/0.2.17/Collins/Api/Logging/Severity
+      #
+      # Compared to Syslog, Collins has an additional level 'note', for an operator input.
+      # We should not come into this level via Genesis.
+      # Anyway, handle it via the default level.
+      priority = case level
```

Comment:
this should be updated to match not with regex, but symbols (`#downcase.#to_sym`).